engine/core/src/main/java/org/datacleaner/job/builder/AnalyzerComponentBuilder.java

```diff
@@ -235,6 +236,21 @@ public AnalyzerJob toAnalyzerJob(boolean validate) throws IllegalStateException
     }
 
     @Override
+    public AnalyzerComponentBuilder<A> removeInputColumn(InputColumn<?> inputColumn,
+            ConfiguredPropertyDescriptor propertyDescriptor) {
+        assert propertyDescriptor.isInputColumn();
+        if (inputColumn == null) {
+            throw new IllegalArgumentException("InputColumn cannot be null");
+        }
+        if (isMultipleJobsDeterminedBy(propertyDescriptor)) {
+            _inputColumns.remove(inputColumn);
```

Comment:
This uses the same pattern as the addXxxx method. For some reason it was missing here and it could easily cause runtime exceptions when removing source columns.